distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Postgres/Postgres_Dialect.enso

```diff
@@ -73,70 +71,70 @@ make_internal_generator_dialect =
    You may need to transform it with `resolve_aggregate` first.
 resolve_target_sql_type aggregate = case aggregate of
     Group_By c _ -> c.sql_type
-    Count _ -> Sql_Type.bigint
-    Count_Distinct _ _ _ -> Sql_Type.bigint
-    Count_Not_Nothing _ _ -> Sql_Type.bigint
-    Count_Nothing _ _ -> Sql_Type.bigint
-    Count_Not_Empty _ _ -> Sql_Type.bigint
-    Count_Empty _ _ -> Sql_Type.bigint
-    Percentile _ _ _ -> Sql_Type.double
+    Count _ -> SQL_Type.bigint
+    Count_Distinct _ _ _ -> SQL_Type.bigint
+    Count_Not_Nothing _ _ -> SQL_Type.bigint
+    Count_Nothing _ _ -> SQL_Type.bigint
+    Count_Not_Empty _ _ -> SQL_Type.bigint
+    Count_Empty _ _ -> SQL_Type.bigint
+    Percentile _ _ _ -> SQL_Type.double
     Mode c _ -> c.sql_type
     First c _ _ _ -> c.sql_type
     Last c _ _ _ -> c.sql_type
     Maximum c _ -> c.sql_type
     Minimum c _ -> c.sql_type
     Shortest c _ -> c.sql_type
     Longest c _ -> c.sql_type
-    Standard_Deviation _ _ _ -> Sql_Type.double
-    Concatenate _ _ _ _ _ _ -> Sql_Type.text
+    Standard_Deviation _ _ _ -> SQL_Type.double
+    Concatenate _ _ _ _ _ _ -> SQL_Type.text
     Sum c _ ->
-        if (c.sql_type == Sql_Type.integer) || (c.sql_type == Sql_Type.smallint) then Sql_Type.bigint else
-            if c.sql_type == Sql_Type.bigint then Sql_Type.numeric else
+        if (c.sql_type == SQL_Type.integer) || (c.sql_type == SQL_Type.smallint) then SQL_Type.bigint else
+            if c.sql_type == SQL_Type.bigint then SQL_Type.numeric else
                 c.sql_type
     Average c _ ->
-        if c.sql_type.is_definitely_integer then Sql_Type.numeric else
-            if c.sql_type.is_definitely_double then Sql_Type.double else
+        if c.sql_type.is_definitely_integer then SQL_Type.numeric else
+            if c.sql_type.is_definitely_double then SQL_Type.double else
                 c.sql_type
-    Median _ _ -> Sql_Type.double
+    Median _ _ -> SQL_Type.double
 
 ## PRIVATE
 agg_count_is_null = Base_Generator.lift_unary_op "COUNT_IS_NULL" arg->
-    Sql.code "COUNT(CASE WHEN " ++ arg.paren ++ Sql.code " IS NULL THEN 1 END)"
+    SQL.code "COUNT(CASE WHEN " ++ arg.paren ++ SQL.code " IS NULL THEN 1 END)"
```

Comment:
While I'm usually not a big fan of unqualified importing of methods (maybe just because I got so used to Enso not supporting it), here I think maybe it would make sense to import `code` unqualified and have a shorter form:

```suggestion
    code "COUNT(CASE WHEN " ++ arg.paren ++ code " IS NULL THEN 1 END)"
```

We use it ALL over the place around here so it would make the functions more concise, and it's an internal module anyway so I don't think we need to worry about scope pollution etc.